config/global.ini.php

```diff
@@ -1245,6 +1245,7 @@
 Plugins[] = Tour
 Plugins[] = PagePerformance
 Plugins[] = CustomDimensions
+Plugins[] = JsTrackerInstallCheck
```

Comment:
We might need to add an update script for this one. On new installations it should be automatically enabled due to this entry, but when updating, the plugin might be disabled.

Reply:
Thank you. That's a good point. I don't think that I've ever dealt with automatically enabling a plugin before. I'll try looking into how to do that.

Reply:
@sgiehl I think I figured out the migration. The command generated the update file for `5.0.0-rc1`, but that's already been released. I manually updated it to `5.0.0-rc2`; is that good or should it be something else?